### PR TITLE
<doc>(server-properties):

### PR DESCRIPTION
**Pull request by ItsKezzer**
- Update generator_settings documentation in server-properties.md

### CODE REVIEW

File: docs/configuration/server-properties.md [hunk 1]
```diff
@@ -305,7 +305,45 @@ environment variable set to `false`, such as
...
+        }
+    ],
+    "biome": "minecraft:desert"
+}
+```
+
+In a docker compose, it would like like the following:
+```yaml
+LEVEL_TYPE: FLAT
+GENERATOR_SETTINGS: '{"layers": [{"block": "minecraft:bedrock","height": 1},{"block": "minecraft:stone","height": 2},{"block": "minecraft:sandstone","height": 3}]},"biome": "minecraft:desert"}'
```
Comment (itzg) [inline docs/configuration/server-properties.md:344]: I was just thinking, a "folded block scalar" might be usable here to keep the JSON looking pretty 

https://yaml-multiline.info/

**Review by itzg — APPROVED** [review]
Thanks, I appreciate you capturing your research here! Let me know if you want to try out the yaml formatting suggestion, but I'm good to merge as-is.

**Comment by ItsKezzer** [comment]
Tested the configuration set as the example, the world generation provided the right result.

**Review by itzg — APPROVED** [review]
Awesome

**Pull request merged**